docs/api.rst

```diff
@@ -171,6 +171,11 @@ Returns all connections for a given object as a ``dict``.
   between objects, e.g., feed, friends, groups, likes, posts. If left empty,
   ``get_connections`` will simply return the authenticated user's basic
   information.
+* ``fields`` – A ``string`` or group of (comma separated) ``strings``. Accepts one 
+    or more fields of the edge specified in connection_name, or one or more 
+    fields/edges of the node connected to the given object_id by said edge. 
+    This parameter allows for nested queries to the GraphAPI, supported by the 
+    field expansion feature. See the GraphAPI docs for syntax and further examples.
```

Comment:
I think a link to the specific section of the Graph API documentation would be useful here.

Reply:
Marty, I completely forgot about this simple request you had. Out of the blue my GitHub account popped in my mind and I have requested a pull with the changes you asked for. Thank you for this super useful SDK!